examples/mgpcg_advanced.py

```diff
@@ -169,12 +206,12 @@ def run(self):
         old_zTr = self.sum[None]
 
         # CG
-        for i in range(400):
+        while max_iters != 0:
```

Comment:
Is there a good reason to use a `while` loop here instead of `for iter in range(max_iters):` ?

Reply:
`max_iter = -1` for no limit.